addons/collectd/0.0.1/install.sh

```diff
@@ -6,7 +6,7 @@ function collectd() {
         case "$LSB_DIST" in
         ubuntu)
             export DEBIAN_FRONTEND=noninteractive
-            dpkg --install --force-depends-version ${src}/ubuntu-${DIST_VERSION}/archives/*.deb
+            dpkg --install --no-install-recommends --force-depends-version ${src}/ubuntu-${DIST_VERSION}/archives/*.deb
```

Comment:
The no install recommends flag should be added to the ubuntu builds so they're not included in the package. https://github.com/replicatedhq/kURL/blob/master/bin/save-manifest-assets.sh#L45

Reply:
But we'll need to check this doesn't break docker, kubelet, and any other debs before adding it there